### PR TITLE
Create report from template

### PR DESCRIPTION
**Pull request by dknochen**

### CODE REVIEW

**Comment by dknochen** [comment]
depending on: https://github.com/28msec/secxbrl.info/pull/505

**Comment by wcandillon** [comment]
@dknochen I improved it a bit. You just need to fix the unrelated regression caused by this new required parameter: https://github.com/28msec/secxbrl.info/commit/20a75c31804ac8be187018c430ec6e77130b5bce#diff-b7a3a2fef3f248c280bac863149d5a86R168

**Comment by dknochen** [comment]
@wcandillon thanks for the improvements. Much more fancy ;). I fixed the tests and made the template selection thumbnails a bit less aggressive (no primary color). Hope it makes sense?

File: app/modules/report-model.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ angular
                    modelOrName !== null) {
             throw new Error('new Report creation with invalid type ' + typeof modelOrName);
         } else if (typeof modelOrName === 'object') {
+            if(modelOrName._id === undefined){
+                modelOrName._id = _uuid();
```
Comment (wcandillon) [inline app/modules/report-model.js]: @dknochen where is this defined?

**Pull request merged**